pkg/certificates/certs.go

```diff
@@ -62,17 +61,15 @@ func createCACertTemplate(expirationInterval time.Duration) (*x509.Certificate,
 	}
 	// Make it into a CA cert and change it so we can use it to sign certs
 	rootCert.IsCA = true
-	rootCert.KeyUsage = x509.KeyUsageCertSign | x509.KeyUsageDigitalSignature
-	rootCert.ExtKeyUsage = []x509.ExtKeyUsage{x509.ExtKeyUsageServerAuth, x509.ExtKeyUsageClientAuth}
+	rootCert.KeyUsage = x509.KeyUsageCertSign
 	rootCert.Subject = pkix.Name{
 		Organization: []string{Organization},
-		CommonName:   "control-plane",
```

Comment:
Does a CA cert need a CommonName?

If not, 👍 . Otherwise, we should probably name it something like "CA".